port/port_posix.h

```diff
@@ -193,7 +193,7 @@ extern void InitOnce(OnceType* once, void (*initializer)());
 #define ALIGN_AS(n) /*empty*/
 #else
 #if defined(__s390__)
-#if defined(__GNUC__) && __GNUC__ < 6
+#if defined(__GNUC__) && __GNUC__ < 7
```

Comment:
why cache line size is changed for version 6?

Reply:
Tbh I can't remember exactly why I chose 6 before as it was some time ago! However, I suspect it was because I was working with earlier gcc compilers in the 4.8.x range and 5.x.x range which showed the issue. I have now found the same issue with a 6.x.x gcc compiler on s390x, so I increased the limit to 7. Newer gcc compilers in the 9.x.x range seem to work just fine, but I don't have the scope to test all versions at the moment.